includes/Admin/Admin_AJAX.php

```diff
@@ -46,6 +50,100 @@ public function get_nonce() {
 		return wp_create_nonce( self::NONCE_KEY );
 	}
 
+	/**
+	 * Handles the AJAX request to setup the runtime environment if needed.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function set_up_environment() {
+		// Verify the nonce before continuing.
+		$valid_nonce = $this->verify_nonce( filter_input( INPUT_POST, 'nonce', FILTER_SANITIZE_STRING ) );
```

Comment:
Noting this for future reference, `FILTER_SANITIZE_STRING` has been deprecated in PHP 8.1 and will begin logging warnings, so we should replace this with a different alternative. See the changelog in: https://www.php.net/manual/en/filter.filters.sanitize.php

Reply:
@joemcgill Can you open a new issue for addressing this throughout the codebase?